lib/matobj2.gd

```diff
@@ -463,7 +463,6 @@ DeclareOperation( "ListOp", [ IsVectorObj, IsFunction ] );
 ##  <Oper Name="Unpack" Arg="v" Label="for a vector object"/>
 ##  <Oper Name="Unpack" Arg="M" Label="for a matrix object"/>
 ##
-##  <Returns>A plain list</Returns>
```

Comment:
The `<Returns>` element is used in several places. What is the reason to remove it here? (I would rather recommend to add such elements than to remove them.)

Reply:
Happy to reinstate this.  It just seemed to me that "Returns: A plain list.  Returns a new mutable plain list ..." was unnecessarily repetitive.